### PR TITLE
52/update dep

### PR DESCRIPTION
**Pull request by pyphilia**
This PR contains only upgrade changes:
- change fastify plugins for the new namenclature
- update all other dependencies
- apply prettier import plugin -> the reason why all the files changes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,37 +29,38 @@
...
-    "eslint": "^8.6.0",
-    "eslint-config-prettier": "^8.3.0",
-    "fastify": "^3.25.3",
+    "@types/jest": "27.5.1",
+    "@types/node": "^17.0.33",
+    "@typescript-eslint/eslint-plugin": "^5.23.0",
+    "@typescript-eslint/parser": "^5.23.0",
+    "eslint": "^8.15.0",
+    "eslint-config-prettier": "^8.5.0",
+    "fastify": "^3.29.0",
```
Comment (dialexo) [inline package.json:55]: Same as above

File: package.json [hunk 2]
```diff
@@ -29,37 +29,38 @@
...
-    "ts-jest": "27.1.2",
-    "ts-node": "10.4.0",
-    "typescript": "^4.0.5"
+    "husky": "8.0.1",
+    "jest": "28.1.0",
+    "prettier": "2.6.2",
+    "slonik": "^28.1.0",
+    "ts-jest": "28.0.2",
+    "ts-node": "10.7.0",
+    "typescript": "^4.6.4"
```
Comment (dialexo) [inline package.json:63]: Same as above

**Review by dialexo — APPROVED** [review]
🍕 Clean! I left a question regarding version strings

File: package.json [hunk 3]
```diff
@@ -29,37 +29,38 @@
...
-    "fastify-auth": "^1.1.0",
-    "fastify-bearer-auth": "^6.1.0",
-    "fastify-cors": "^6.0.2",
-    "fastify-jwt": "^4.1.3",
-    "fluent-json-schema": "^3.0.1",
+    "@fastify/auth": "2.0.0",
+    "@fastify/bearer-auth": "7.0.1",
+    "@fastify/cors": "7.0.0",
+    "@fastify/jwt": "5.0.1",
+    "fluent-json-schema": "^3.1.0",
```
[the remark this answers is not in the document]
Comment (pyphilia) [reply package.json:36]: Alright, I'll let you discuss with JC ;) I'll set the specific version menawhile.

**Pull request merged**